app/controllers/api/v1/artists_controller.rb

```diff
@@ -15,6 +15,7 @@ def create
 
   def show
     artist = Artist.find(params[:id])
+    review = Review.new
     render json: artist
```

Comment:
you arent doing anything with this review object! In an api controller, we dont need to prepare an empty review. delete line 18